lib/main_common.pm

```diff
@@ -458,7 +458,9 @@ sub load_extra_tests() {
         # dependency of git test
         loadtest "console/sshd";
         loadtest "console/zypper_ref";
-        loadtest "console/zypper_info";
+        if (sle_version_at_least('12-SP2') || check_env('DISTRI', 'opensuse')) {
```

Comment:
it test funkcionality  included in SLE12SP2+ so why run on older, But changes before disabled half of test because wrong repository handling ...

So this change disables on SLE12SP1 and older , and enables this test to work on released products ..

In another way , original change only moves problem to FUTURE aftere release SLE12SP3 ..